### PR TITLE
enh(systemtags): Add accessible system tags form

### PR DESCRIPTION
**Pull request by Pytal**
- For https://github.com/nextcloud/server/issues/41208

## Summary

| | Screenshot |
--- | ---
**Before** | image
**After** | image

## To Do

- [x] Tests

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Comment by JuliaKirschenheuter** [comment]
😱 would need time to review

**Review by ShGKme — APPROVED** [review]
Works fine.

But from a11y perspective, I'm not sure that this is clear when this form is at the same time create and edit form, especially when searching a tag to edit is a part of the form.

**Comment by Pytal** [comment]
> But from a11y perspective, I'm not sure that this is clear when this form is at the same time create and edit form, especially when searching a tag to edit is a part of the form.

Will confirm with @michaelnissenbaum

File: apps/systemtags/src/views/SystemTagsSection.vue [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+		t,
+
+		handleCreate(tag: TagWithId) {
+			this.tags.unshift(tag)
+		},
+
+		handleUpdate(tag: TagWithId) {
+			const tagIndex = this.tags.findIndex(currTag => currTag.id === tag.id)
+			this.tags.splice(tagIndex, 1)
+			this.tags.unshift(tag)
```
[the remark this answers is not in the document]
Comment (Pytal) [reply apps/systemtags/src/views/SystemTagsSection.vue:90]: Yes, this is to move the most recently updated/created tags to the top

**Review by susnux — APPROVED** [review]

**Review by JuliaKirschenheuter — APPROVED** [review]

**Pull request merged**